js-foundations.md

```diff
@@ -15,66 +15,80 @@ logIt()
 
 1a. Look at this Javascript function and try to predict what the console show.
 
+text = 'inside','outside'
 
 
 1b. Test the function. Explain why the function returned what it did.
 
-  Your answer:
+  Your answer: I Tested it and all it said was undefined.
 
-  Researched answer:
+  Researched answer: I Tested it and all it said was undefined.
 
```

Comment:
Yup, that is the weird part of this question. When the console.log is called the variable text has been declared but not defined according to JavaScript.